ext/k3.cc

```diff
@@ -96,7 +96,7 @@ int main(int argc, char *argv[]) {
 #ifdef HAVE_CUDA
     fprintf(stdout, "Cuda enabled\n");
     CuDevice &cu_device = CuDevice::Instantiate();
-    cu_device.SetVerbose(true);
+    //cu_device.SetVerbose(true);
```

Comment:
Same line?

Reply:
This was in the instructions in that blog post. I tried it without commenting it out and the build breaks. You can't use `cu_device.SetVerbose(true);`.